layers/+completion/helm/local/helm-spacemacs-help/helm-spacemacs-help.el

```diff
@@ -192,12 +192,12 @@
 (defvar helm-spacemacs-help--layer-map
   (let ((map (make-sparse-keymap)))
     (set-keymap-parent map helm-map)
-    (define-key map (kbd "<S-return>") '(lambda () (interactive)
-                                          ;; Add Layer
-                                          (helm-select-nth-action 3)))
-    (define-key map (kbd "<M-return>") '(lambda () (interactive)
-                                          ;; Open packages.el
-                                          (helm-select-nth-action 1)))
+    (define-key map (kbd "<S-return>") (lambda ()
+                                         "Install a layer, the current Helm candidate."
+                                         (interactive) (helm-select-nth-action 5)))
```

Comment:
why is it changed from 3 to 5?

Reply:
The actions are zero indexed. 3 was wrong, given the comment that did exist which was to "add layer", meaning install it.